src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
Thanks @calbach . Just so I'm clear, the way I understand it is we want to retrieve all the reads in an interval (start, end) across ReadGroupSets [s1, s2, ..., sn]. Can the client not achieve this by making individual requests for (start, end) in s1,s2, ..., sn? Unless the number of reads we hit in each set is tiny, the extra overhead of this approach would be negligible.